src/leader.py

```diff
@@ -62,12 +62,17 @@ def turn_right(self):
     def turn_straight(self):
         self.fw.turn(self.STRAIGHT_ANGLE)
 
-
-    def turn(self,position):
-        if position == -1:
-            self.turn_left()
-        elif position == 1:
-            self.turn_right()
+    def set_turn_angle(self,position):
+        self.fw.turn(self.STRAIGHT_ANGLE + 45  * position)
```

Comment:
Using magic numbers is not allowed. Why are you adding 45 to your turn?

Reply:
also PEMDAS. Is this doing what you want? Maybe wrap it in parentheses to make it clearer.